frontend/src/components/TaskCreator/index.tsx

```diff
@@ -48,11 +51,22 @@ const initialState = (): State => ({
   needToSwitchFocus: false,
 });
 
-export default class TaskCreator extends React.PureComponent<{}, State> {
+export class TaskCreator extends React.PureComponent<Props, State> {
   public readonly state: State = initialState();
 
   private addTask: HTMLInputElement | null | undefined;
 
+  private darkModeStyles: undefined | CSSProperties;
+
+  constructor(props: {theme: Theme}) {
+    super(props);
+    const { theme } = props;
+    this.darkModeStyles = theme === 'dark' ? {
+      background: 'black',
+      color: 'white',
+    } : undefined;
```

Comment:
`darkModeStyles ` can be out-of-sync with the props if you wrote it in this way.